contracts/TimelockVaultVotes.sol

```diff
@@ -0,0 +1,207 @@
+// SPDX-License-Identifier: MIT
```

Comment:
generally I would split this commit into two: 1) copy the existing contact and rename without any single change except the name 2) made new changes
- this is would review a lot because of info what was exactly changed. Thanks

Reply:
Yep, totally agree. Would implement this practice in the future.